<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>alevinQC • alevinQC</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.7/cosmo/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script><!-- sticky kit --><script src="https://cdnjs.cloudflare.com/ajax/libs/sticky-kit/1.1.3/sticky-kit.min.js" integrity="sha256-c4Rlo1ZozqTPE2RLuvbusY3+SU1pQaJC0TjuhygMipw=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="alevinQC">
<meta property="og:description" content="">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">alevinQC</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.99.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/alevinqc.html">alevinQC</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/csoneson/alevinQC">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      
      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1>alevinQC</h1>
                        <h4 class="author">Charlotte Soneson</h4>
            
            <h4 class="date">2019-02-01</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/csoneson/alevinQC/blob/master/vignettes/alevinqc.Rmd"><code>vignettes/alevinqc.Rmd</code></a></small>
      <div class="hidden name"><code>alevinqc.Rmd</code></div>

    </div>

    
        <div class="abstract">
      <p class="abstract">Abstract</p>
      <em><a href="https://bioconductor.org/packages/3.9/alevinQC">alevinQC</a></em> reads output files from alevin and generates summary reports.
    </div>
    
<div id="introduction" class="section level1">
<h1 class="hasAnchor">
<a href="#introduction" class="anchor"></a>Introduction</h1>
<p>The purpose of the <em><a href="https://bioconductor.org/packages/3.9/alevinQC">alevinQC</a></em> package is to generate a summary QC report based on the output of an <a href="https://salmon.readthedocs.io/en/latest/alevin.html">alevin</a> <span class="citation">(Srivastava et al. 2018)</span> run. The QC report can be generated as a html or pdf file, or launched as a shiny application.</p>
</div>
<div id="installation" class="section level1">
<h1 class="hasAnchor">
<a href="#installation" class="anchor"></a>Installation</h1>
<p>Currently, <code>alevinQC</code> can be installed from GitHub, using e.g. the <code>BiocManager</code> CRAN package.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="cf">if</span> (<span class="op">!</span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/ns-load">requireNamespace</a></span>(<span class="st">"BiocManager"</span>, <span class="dt">quietly =</span> <span class="ot">TRUE</span>))</a>
<a class="sourceLine" id="cb1-2" data-line-number="2">    <span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/install.packages">install.packages</a></span>(<span class="st">"BiocManager"</span>)</a>
<a class="sourceLine" id="cb1-3" data-line-number="3">BiocManager<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/BiocManager/topics/install">install</a></span>(<span class="st">"csoneson/alevinQC"</span>)</a></code></pre></div>
<p>After installation, load the package into the R session.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb2-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(alevinQC)</a></code></pre></div>
</div>
<div id="assumed-output-directory-structure" class="section level1">
<h1 class="hasAnchor">
<a href="#assumed-output-directory-structure" class="anchor"></a>Assumed output directory structure</h1>
<p>For more information about running alevin, we refer to the <a href="https://salmon.readthedocs.io/en/latest/alevin.html">documentation</a>. When invoked, alevin generates several output files in the specified output directory. <em><a href="https://bioconductor.org/packages/3.9/alevinQC">alevinQC</a></em> assumes that this structure is retained, and will return an error if it isn’t - thus, it is not recommended to move or rename the output files from alevin. <em><a href="https://bioconductor.org/packages/3.9/alevinQC">alevinQC</a></em> assumes that the following files (in the indicated structure) are available in the provided <code>baseDir</code> (note that currently, in order to generate the full set of files, alevin must be invoked with the <code>--dumpFeatures</code> flag).</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb3-1" data-line-number="1">baseDir</a>
<a class="sourceLine" id="cb3-2" data-line-number="2">  <span class="op">|--</span>alevin</a>
<a class="sourceLine" id="cb3-3" data-line-number="3">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>featureDump.txt</a>
<a class="sourceLine" id="cb3-4" data-line-number="4">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>filtered_cb_frequency.txt</a>
<a class="sourceLine" id="cb3-5" data-line-number="5">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>MappedUmi.txt</a>
<a class="sourceLine" id="cb3-6" data-line-number="6">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>quants_mat_cols.txt</a>
<a class="sourceLine" id="cb3-7" data-line-number="7">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>quants_mat_rows.txt</a>
<a class="sourceLine" id="cb3-8" data-line-number="8">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>quants_mat.gz</a>
<a class="sourceLine" id="cb3-9" data-line-number="9">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>raw_cb_frequency.txt</a>
<a class="sourceLine" id="cb3-10" data-line-number="10">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>whitelist.txt</a>
<a class="sourceLine" id="cb3-11" data-line-number="11">  <span class="op">|--</span>aux_info</a>
<a class="sourceLine" id="cb3-12" data-line-number="12">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>meta_info.json</a>
<a class="sourceLine" id="cb3-13" data-line-number="13">  <span class="op">|--</span>cmd_info.json</a></code></pre></div>
</div>
<div id="check-that-all-required-alevin-files-are-available" class="section level1">
<h1 class="hasAnchor">
<a href="#check-that-all-required-alevin-files-are-available" class="anchor"></a>Check that all required alevin files are available</h1>
<p>The report generation functions (see below) will check that all the required files are available in the provided base directory. However, you can also call the function <code><a href="../reference/checkAlevinInputFiles.html">checkAlevinInputFiles()</a></code> to run the check manually. If one or more files are missing, the function will raise an error indicating the missing file(s).</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb4-1" data-line-number="1">baseDir &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/system.file">system.file</a></span>(<span class="st">"extdata/alevin/neurons_900"</span>, <span class="dt">package =</span> <span class="st">"tximportData"</span>)</a>
<a class="sourceLine" id="cb4-2" data-line-number="2"><span class="kw"><a href="../reference/checkAlevinInputFiles.html">checkAlevinInputFiles</a></span>(<span class="dt">baseDir =</span> baseDir)</a></code></pre></div>
</div>
<div id="generate-qc-report" class="section level1">
<h1 class="hasAnchor">
<a href="#generate-qc-report" class="anchor"></a>Generate QC report</h1>
<p>The <code><a href="../reference/alevinQCReport.html">alevinQCReport()</a></code> function generates the QC report from the alevin output. Depending on the file extension of the <code>outputFile</code> argument, and the value of <code>outputFormat</code>, the function can generate either an html report or a pdf report.</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb5-1" data-line-number="1">outputDir &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/tempfile">tempdir</a></span>()</a>
<a class="sourceLine" id="cb5-2" data-line-number="2"><span class="kw"><a href="../reference/alevinQCReport.html">alevinQCReport</a></span>(<span class="dt">baseDir =</span> baseDir, <span class="dt">sampleId =</span> <span class="st">"testSample"</span>, </a>
<a class="sourceLine" id="cb5-3" data-line-number="3">               <span class="dt">outputFile =</span> <span class="st">"alevinReport.html"</span>, </a>
<a class="sourceLine" id="cb5-4" data-line-number="4">               <span class="dt">outputFormat =</span> <span class="st">"html_document"</span>,</a>
<a class="sourceLine" id="cb5-5" data-line-number="5">               <span class="dt">outputDir =</span> outputDir, <span class="dt">forceOverwrite =</span> <span class="ot">TRUE</span>)</a></code></pre></div>
</div>
<div id="create-shiny-app" class="section level1">
<h1 class="hasAnchor">
<a href="#create-shiny-app" class="anchor"></a>Create shiny app</h1>
<p>In addition to static reports, <em><a href="https://bioconductor.org/packages/3.9/alevinQC">alevinQC</a></em> can also generate a shiny application, containing the same summary figures as the pdf and html reports.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb6-1" data-line-number="1">app &lt;-<span class="st"> </span><span class="kw"><a href="../reference/alevinQCShiny.html">alevinQCShiny</a></span>(<span class="dt">baseDir =</span> baseDir, <span class="dt">sampleId =</span> <span class="st">"testSample"</span>)</a></code></pre></div>
<p>Once created, the app can be launched using the <code>runApp()</code> function from the <em><a href="https://CRAN.R-project.org/package=shiny">shiny</a></em> package.</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb7-1" data-line-number="1">shiny<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/shiny/topics/runApp">runApp</a></span>(app)</a></code></pre></div>
</div>
<div id="generate-individual-plots" class="section level1">
<h1 class="hasAnchor">
<a href="#generate-individual-plots" class="anchor"></a>Generate individual plots</h1>
<p>The individual plots included in the QC reports can also be independently generated. To do so, we must first read the alevin output into an R object.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb8-1" data-line-number="1">alevin &lt;-<span class="st"> </span><span class="kw"><a href="../reference/readAlevinQC.html">readAlevinQC</a></span>(<span class="dt">baseDir =</span> baseDir)</a>
<a class="sourceLine" id="cb8-2" data-line-number="2"><span class="co">#&gt; reading in alevin gene-level counts and inferential variance across cells</span></a>
<a class="sourceLine" id="cb8-3" data-line-number="3"><span class="co">#&gt; Joining, by = "CB"</span></a></code></pre></div>
<p>The resulting list contains three entries:</p>
<ul>
<li>
<code>cbTable</code>: a <code>data.frame</code> with various inferred characteristics of the individual cell barcodes</li>
<li>
<code>summaryTables</code>: a list of <code>data.frame</code>s with summary information about the full data set, the initial set of whitelisted cells and the final set of whitelisted cells, respectively</li>
<li>
<code>versionTable</code>: a <code>matrix</code> with information about the invokation of alevin</li>
</ul>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb9-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(alevin<span class="op">$</span>cbTable)</a>
<a class="sourceLine" id="cb9-2" data-line-number="2"><span class="co">#&gt;                 CB originalFreq ranking collapsedFreq mappingRate</span></a>
<a class="sourceLine" id="cb9-3" data-line-number="3"><span class="co">#&gt; 1 TGGTTCCAGTCCATAC       294486       1        299998    0.705001</span></a>
<a class="sourceLine" id="cb9-4" data-line-number="4"><span class="co">#&gt; 2 TGCTACCCACCAGCAC       236420       2        244868    0.698152</span></a>
<a class="sourceLine" id="cb9-5" data-line-number="5"><span class="co">#&gt; 3 TTCTTAGGTGAAGGCT       215630       3        219506    0.713876</span></a>
<a class="sourceLine" id="cb9-6" data-line-number="6"><span class="co">#&gt; 4 TGATTTCCAATAACGA       208554       4        210984    0.715746</span></a>
<a class="sourceLine" id="cb9-7" data-line-number="7"><span class="co">#&gt; 5 ATGCGATTCACCCTCA       207135       5        211894    0.641656</span></a>
<a class="sourceLine" id="cb9-8" data-line-number="8"><span class="co">#&gt; 6 CACACAAAGTCAAGCG       189231       6        193700    0.699179</span></a>
<a class="sourceLine" id="cb9-9" data-line-number="9"><span class="co">#&gt;   duplicationRate dedupRate nbrGenesAboveMean nbrMappedUMI totalUMICount</span></a>
<a class="sourceLine" id="cb9-10" data-line-number="10"><span class="co">#&gt; 1      0.00207650  0.591592              5963       211499      86377.79</span></a>
<a class="sourceLine" id="cb9-11" data-line-number="11"><span class="co">#&gt; 2      0.00232080  0.599421              5431       170955      68480.92</span></a>
<a class="sourceLine" id="cb9-12" data-line-number="12"><span class="co">#&gt; 3      0.00141115  0.546762              5336       156700      71022.42</span></a>
<a class="sourceLine" id="cb9-13" data-line-number="13"><span class="co">#&gt; 4      0.00140541  0.580716              4974       151011      63316.44</span></a>
<a class="sourceLine" id="cb9-14" data-line-number="14"><span class="co">#&gt; 5      0.00107668  0.584400              5168       135963      56506.21</span></a>
<a class="sourceLine" id="cb9-15" data-line-number="15"><span class="co">#&gt; 6      0.00137204  0.595003              4822       135431      54849.20</span></a>
<a class="sourceLine" id="cb9-16" data-line-number="16"><span class="co">#&gt;   nbrGenesAboveZero inFinalWhiteList inFirstWhiteList</span></a>
<a class="sourceLine" id="cb9-17" data-line-number="17"><span class="co">#&gt; 1              8779             TRUE             TRUE</span></a>
<a class="sourceLine" id="cb9-18" data-line-number="18"><span class="co">#&gt; 2              8199             TRUE             TRUE</span></a>
<a class="sourceLine" id="cb9-19" data-line-number="19"><span class="co">#&gt; 3              8088             TRUE             TRUE</span></a>
<a class="sourceLine" id="cb9-20" data-line-number="20"><span class="co">#&gt; 4              7716             TRUE             TRUE</span></a>
<a class="sourceLine" id="cb9-21" data-line-number="21"><span class="co">#&gt; 5              7841             TRUE             TRUE</span></a>
<a class="sourceLine" id="cb9-22" data-line-number="22"><span class="co">#&gt; 6              7478             TRUE             TRUE</span></a></code></pre></div>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb10-1" data-line-number="1">knitr<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/knitr/topics/kable">kable</a></span>(alevin<span class="op">$</span>summaryTables<span class="op">$</span>fullDataset)</a></code></pre></div>
<table class="table"><tbody>
<tr class="odd">
<td align="left">Total number of processed reads</td>
<td align="left">52805264</td>
</tr>
<tr class="even">
<td align="left">Number of reads with valid cell barcode (no Ns)</td>
<td align="left">52653685</td>
</tr>
<tr class="odd">
<td align="left">Total number of observed cell barcodes</td>
<td align="left">657180</td>
</tr>
</tbody></table>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb11-1" data-line-number="1">knitr<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/knitr/topics/kable">kable</a></span>(alevin<span class="op">$</span>summaryTables<span class="op">$</span>initialWhitelist)</a></code></pre></div>
<table class="table"><tbody>
<tr class="odd">
<td align="left">Number of barcodes in initial whitelist</td>
<td align="left">2138</td>
</tr>
<tr class="even">
<td align="left">Fraction reads in initial whitelist barcodes</td>
<td align="left">88.93%</td>
</tr>
<tr class="odd">
<td align="left">Mean number of reads per cell (initial whitelist)</td>
<td align="left">21901</td>
</tr>
<tr class="even">
<td align="left">Median number of reads per cell (initial whitelist)</td>
<td align="left">12868</td>
</tr>
<tr class="odd">
<td align="left">Median number of detected genes per cell (initial whitelist)</td>
<td align="left">1362</td>
</tr>
<tr class="even">
<td align="left">Total number of detected genes (initial whitelist)</td>
<td align="left">22415</td>
</tr>
<tr class="odd">
<td align="left">Median UMI count per cell (initial whitelist)</td>
<td align="left">3073</td>
</tr>
</tbody></table>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb12-1" data-line-number="1">knitr<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/knitr/topics/kable">kable</a></span>(alevin<span class="op">$</span>summaryTables<span class="op">$</span>finalWhitelist)</a></code></pre></div>
<table class="table"><tbody>
<tr class="odd">
<td align="left">Number of barcodes in final whitelist</td>
<td align="left">1302</td>
</tr>
<tr class="even">
<td align="left">Fraction reads in final whitelist barcodes</td>
<td align="left">85.52%</td>
</tr>
<tr class="odd">
<td align="left">Mean number of reads per cell (final whitelist)</td>
<td align="left">34584</td>
</tr>
<tr class="even">
<td align="left">Median number of reads per cell (final whitelist)</td>
<td align="left">25712</td>
</tr>
<tr class="odd">
<td align="left">Median number of detected genes per cell (final whitelist)</td>
<td align="left">2506</td>
</tr>
<tr class="even">
<td align="left">Total number of detected genes (final whitelist)</td>
<td align="left">22286</td>
</tr>
<tr class="odd">
<td align="left">Median UMI count per cell (final whitelist)</td>
<td align="left">6807</td>
</tr>
</tbody></table>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb13-1" data-line-number="1">knitr<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/knitr/topics/kable">kable</a></span>(alevin<span class="op">$</span>versionTable)</a></code></pre></div>
<table class="table"><tbody>
<tr class="odd">
<td align="left">Start time</td>
<td align="left">Thu Jan 24 12:13:05 2019</td>
</tr>
<tr class="even">
<td align="left">Salmon version</td>
<td align="left">0.12.1</td>
</tr>
<tr class="odd">
<td align="left">Index</td>
<td align="left">/mnt/scratch6/avi/data/cgat/index/salmon/mm_transcriptome.salmon.index</td>
</tr>
<tr class="even">
<td align="left">R1file</td>
<td align="left">/mnt/scratch6/avi/data/cgat/fastqs/neurons_900.fastq.1.gz</td>
</tr>
<tr class="odd">
<td align="left">R2file</td>
<td align="left">/mnt/scratch6/avi/data/cgat/fastqs/neurons_900.fastq.2.gz</td>
</tr>
<tr class="even">
<td align="left">tgMap</td>
<td align="left">/mnt/scratch6/avi/data/cgat/references/metadata/mm_t2g.tsv</td>
</tr>
</tbody></table>
<p>The plots can now be generated using the dedicated plotting functions provided with <em><a href="https://bioconductor.org/packages/3.9/alevinQC">alevinQC</a></em> (see the help file for the respective function for more information).</p>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb14-1" data-line-number="1"><span class="kw"><a href="../reference/plotAlevinKneeRaw.html">plotAlevinKneeRaw</a></span>(alevin<span class="op">$</span>cbTable)</a></code></pre></div>
<p><img src="alevinqc_files/figure-html/v13-1.png" width="700"></p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb15-1" data-line-number="1"><span class="kw"><a href="../reference/plotAlevinBarcodeCollapse.html">plotAlevinBarcodeCollapse</a></span>(alevin<span class="op">$</span>cbTable)</a></code></pre></div>
<p><img src="alevinqc_files/figure-html/v13-2.png" width="700"></p>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb16-1" data-line-number="1"><span class="kw"><a href="../reference/plotAlevinQuant.html">plotAlevinQuant</a></span>(alevin<span class="op">$</span>cbTable)</a></code></pre></div>
<p><img src="alevinqc_files/figure-html/v13-3.png" width="700"></p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb17-1" data-line-number="1"><span class="kw"><a href="../reference/plotAlevinKneeNbrGenes.html">plotAlevinKneeNbrGenes</a></span>(alevin<span class="op">$</span>cbTable)</a></code></pre></div>
<p><img src="alevinqc_files/figure-html/v13-4.png" width="700"></p>
</div>
<div id="session-info" class="section level1">
<h1 class="hasAnchor">
<a href="#session-info" class="anchor"></a>Session info</h1>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb18-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/sessionInfo">sessionInfo</a></span>()</a>
<a class="sourceLine" id="cb18-2" data-line-number="2"><span class="co">#&gt; R Under development (unstable) (2018-11-05 r75543)</span></a>
<a class="sourceLine" id="cb18-3" data-line-number="3"><span class="co">#&gt; Platform: x86_64-apple-darwin15.6.0 (64-bit)</span></a>
<a class="sourceLine" id="cb18-4" data-line-number="4"><span class="co">#&gt; Running under: macOS High Sierra 10.13.6</span></a>
<a class="sourceLine" id="cb18-5" data-line-number="5"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb18-6" data-line-number="6"><span class="co">#&gt; Matrix products: default</span></a>
<a class="sourceLine" id="cb18-7" data-line-number="7"><span class="co">#&gt; BLAS: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRblas.0.dylib</span></a>
<a class="sourceLine" id="cb18-8" data-line-number="8"><span class="co">#&gt; LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib</span></a>
<a class="sourceLine" id="cb18-9" data-line-number="9"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb18-10" data-line-number="10"><span class="co">#&gt; locale:</span></a>
<a class="sourceLine" id="cb18-11" data-line-number="11"><span class="co">#&gt; [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8</span></a>
<a class="sourceLine" id="cb18-12" data-line-number="12"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb18-13" data-line-number="13"><span class="co">#&gt; attached base packages:</span></a>
<a class="sourceLine" id="cb18-14" data-line-number="14"><span class="co">#&gt; [1] stats     graphics  grDevices utils     datasets  methods   base     </span></a>
<a class="sourceLine" id="cb18-15" data-line-number="15"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb18-16" data-line-number="16"><span class="co">#&gt; other attached packages:</span></a>
<a class="sourceLine" id="cb18-17" data-line-number="17"><span class="co">#&gt; [1] bindrcpp_0.2.2   alevinQC_0.99.0  BiocStyle_2.11.0</span></a>
<a class="sourceLine" id="cb18-18" data-line-number="18"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb18-19" data-line-number="19"><span class="co">#&gt; loaded via a namespace (and not attached):</span></a>
<a class="sourceLine" id="cb18-20" data-line-number="20"><span class="co">#&gt;  [1] tximport_1.11.6      tidyselect_0.2.5     xfun_0.4            </span></a>
<a class="sourceLine" id="cb18-21" data-line-number="21"><span class="co">#&gt;  [4] purrr_0.3.0          colorspace_1.4-0     htmltools_0.3.6     </span></a>
<a class="sourceLine" id="cb18-22" data-line-number="22"><span class="co">#&gt;  [7] yaml_2.2.0           rlang_0.3.1          pkgdown_1.3.0.9000  </span></a>
<a class="sourceLine" id="cb18-23" data-line-number="23"><span class="co">#&gt; [10] pillar_1.3.1         later_0.7.5          glue_1.3.0          </span></a>
<a class="sourceLine" id="cb18-24" data-line-number="24"><span class="co">#&gt; [13] RColorBrewer_1.1-2   bindr_0.1.1          plyr_1.8.4          </span></a>
<a class="sourceLine" id="cb18-25" data-line-number="25"><span class="co">#&gt; [16] stringr_1.3.1        munsell_0.5.0        commonmark_1.7      </span></a>
<a class="sourceLine" id="cb18-26" data-line-number="26"><span class="co">#&gt; [19] gtable_0.2.0         htmlwidgets_1.3      memoise_1.1.0       </span></a>
<a class="sourceLine" id="cb18-27" data-line-number="27"><span class="co">#&gt; [22] evaluate_0.12        labeling_0.3         knitr_1.21          </span></a>
<a class="sourceLine" id="cb18-28" data-line-number="28"><span class="co">#&gt; [25] GGally_1.4.0         crosstalk_1.0.0      httpuv_1.4.5.1      </span></a>
<a class="sourceLine" id="cb18-29" data-line-number="29"><span class="co">#&gt; [28] highr_0.7            Rcpp_1.0.0           xtable_1.8-3        </span></a>
<a class="sourceLine" id="cb18-30" data-line-number="30"><span class="co">#&gt; [31] backports_1.1.3      scales_1.0.0         promises_1.0.1      </span></a>
<a class="sourceLine" id="cb18-31" data-line-number="31"><span class="co">#&gt; [34] DT_0.5               BiocManager_1.30.4   desc_1.2.0          </span></a>
<a class="sourceLine" id="cb18-32" data-line-number="32"><span class="co">#&gt; [37] mime_0.6             fs_1.2.6             rjson_0.2.20        </span></a>
<a class="sourceLine" id="cb18-33" data-line-number="33"><span class="co">#&gt; [40] ggplot2_3.1.0        digest_0.6.18        stringi_1.2.4       </span></a>
<a class="sourceLine" id="cb18-34" data-line-number="34"><span class="co">#&gt; [43] bookdown_0.9         dplyr_0.7.8          shiny_1.2.0         </span></a>
<a class="sourceLine" id="cb18-35" data-line-number="35"><span class="co">#&gt; [46] rprojroot_1.3-2      grid_3.6.0           cowplot_0.9.4       </span></a>
<a class="sourceLine" id="cb18-36" data-line-number="36"><span class="co">#&gt; [49] tools_3.6.0          magrittr_1.5         lazyeval_0.2.1      </span></a>
<a class="sourceLine" id="cb18-37" data-line-number="37"><span class="co">#&gt; [52] tibble_2.0.1         crayon_1.3.4         pkgconfig_2.0.2     </span></a>
<a class="sourceLine" id="cb18-38" data-line-number="38"><span class="co">#&gt; [55] MASS_7.3-51.1        xml2_1.2.0           shinydashboard_0.7.1</span></a>
<a class="sourceLine" id="cb18-39" data-line-number="39"><span class="co">#&gt; [58] assertthat_0.2.0     rmarkdown_1.11       reshape_0.8.8       </span></a>
<a class="sourceLine" id="cb18-40" data-line-number="40"><span class="co">#&gt; [61] roxygen2_6.1.1       rstudioapi_0.9.0     R6_2.3.0            </span></a>
<a class="sourceLine" id="cb18-41" data-line-number="41"><span class="co">#&gt; [64] compiler_3.6.0</span></a></code></pre></div>
</div>
<div id="references" class="section level1 unnumbered">
<h1 class="hasAnchor">
<a href="#references" class="anchor"></a>References</h1>
<div id="refs" class="references">
<div id="ref-Srivastava2018-fz">
<p>Srivastava, Avi, Laraib Malik, Tom Sean Smith, Ian Sudbery, and Rob Patro. 2018. “Alevin Efficiently Estimates Accurate Gene Abundances from dscRNA-seq Data.” <em>bioRxiv Doi:10.1101/335000</em>.</p>
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
        <div id="tocnav">
      <h2 class="hasAnchor">
<a href="#tocnav" class="anchor"></a>Contents</h2>
      <ul class="nav nav-pills nav-stacked">
<li><a href="#introduction">Introduction</a></li>
      <li><a href="#installation">Installation</a></li>
      <li><a href="#assumed-output-directory-structure">Assumed output directory structure</a></li>
      <li><a href="#check-that-all-required-alevin-files-are-available">Check that all required alevin files are available</a></li>
      <li><a href="#generate-qc-report">Generate QC report</a></li>
      <li><a href="#create-shiny-app">Create shiny app</a></li>
      <li><a href="#generate-individual-plots">Generate individual plots</a></li>
      <li><a href="#session-info">Session info</a></li>
      <li><a href="#references">References</a></li>
      </ul>
</div>
      </div>

</div>


      <footer><div class="copyright">
  <p>Developed by <a href="https://csoneson.github.io">Charlotte Soneson</a>, Avi Srivastava.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.3.0.9000.</p>
</div>
      </footer>
</div>

  

  </body>
</html>
